src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOARD = 101;
+    private static final int RADIUS = 51;
+    private static final int SET_OF_FIGURES = 5;
```

Comment:
The constant names `BOARD` and `RADIUS` are not descriptive enough to understand their purpose in the context of figure generation. Consider renaming them to reflect their usage, such as `MAX_SIDE_LENGTH` and `MAX_RADIUS`, respectively.